js/components/Chips/ChipsContainer.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import update from "immutability-helper";
+import Chips from "./Chips";
+
+import { styles } from "./styles";
+
+export default class ChipsContainer extends Component {
+  constructor() {
```

Comment:
if you are receiving props don't forget to pass "props" in your constructor and in your super.